src/main/Config.cpp

```diff
@@ -237,20 +237,30 @@ Config::loadQset(std::shared_ptr<cpptoml::toml_group> group, SCPQuorumSet& qset,
 }
 
 void
-Config::load(std::string const& filename)
+Config::load(std::string fileName)
```

Comment:
I was able to revert that change. Making this into static function will be possible after removing DeprecatedCommandLine - for now it would be more work that it deserves (because of Config wariable being used both inside and outside of catch block and me not wanting to do Config cfg; cfg = Config::load();).